config/database.js

```diff
@@ -0,0 +1,9 @@
+import mongoose from 'mongoose'
+
+export default function db() {
+  mongoose.connect('mongodb://127.0.0.1:27017/smart-offer', {
```

Comment:
It is ok for now to have this hardcode string, but remember we need to add support and use `environment variables`

Reply:
Fixed, now I am using environment variable